processing/src/test/java/org/apache/druid/query/groupby/UnnestGroupByQueryRunnerTest.java

```diff
@@ -699,8 +696,7 @@ public void testGroupByOnUnnestedVirtualMultiColumn()
     final DataSource unnestDataSource = UnnestDataSource.create(
         new TableDataSource(QueryRunnerTestHelper.DATA_SOURCE),
         "vc",
-        QueryRunnerTestHelper.PLACEMENTISH_DIMENSION_UNNEST,
-        null
+        QueryRunnerTestHelper.PLACEMENTISH_DIMENSION_UNNEST
```

Comment:
Sorry, what is a `PLACEMENTISH`?

Reply:
It is a dimension in our test setup that is of the format `placement,<some_string>`